### PR TITLE
Fix: No local URL rewrite

### PR DESCRIPTION
**Pull request by stramel**
Modified to run headless and add code coverage checks

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,9 +12,9 @@ install:
   - npm install -g polymer-cli
   - polymer install --variants
 before_script:
-  - polymer lint --rules polymer-2-hybrid --input *.html
+  - polymer lint --input *.html
```
Comment (admwx7) [inline .travis.yml:15]: Not running eslint as well?

File: mtz-url-rewrite-interceptor.html [hunk 2]
```diff
@@ -111,7 +111,7 @@
         isLocal = this.__isLocal, localBaseUrl = this.localBaseUrl, baseUrl = this.baseUrl,
         pattern = this.pattern, replacer = this.replacer, replacement = this.replacement, origin = this.__origin) {
         // Determine baseUrl
-        baseUrl = isLocal ? localBaseUrl : baseUrl || origin;
+        baseUrl = isLocal && localBaseUrl ? localBaseUrl : baseUrl || origin;
```
Comment (admwx7) [inline mtz-url-rewrite-interceptor.html]: You need to add `()` to scope this and ensure it's running in the correct order.  The op order in this case is going to be `&&` > `||` > `... ? ... : ...` which equates to:
`baseUrl = `(isLocal && localBaseUrl) ? localBaseUrl : (baseUrl || origin);` which doesn't seem right to me.

If you're curious about order
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence

File: package.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  "name": "@mtz/mtz-ajax-interceptor",
+  "version": "0.5.0",
+  "description": "A group of elements to support intercepting ajax requests for injecting headers or handling responses globally.",
+  "main": "all-imports.html",
+  "directories": {
+    "example": "demo",
+    "test": "test"
+  },
+  "scripts": {
+    "test": "wct"
```
Comment (admwx7) [inline package.json:11]: `polymer test` doesn't work for this?  It should be using WCT internally.

File: .travis.yml [hunk 4]
```diff
@@ -12,9 +12,9 @@ install:
   - npm install -g polymer-cli
```
Comment (admwx7) [inline .travis.yml:12]: You've included deps in a package.json, do you also need to run `npm i` here?

**Review by admwx7 — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (stramel) [reply package.json:11]: Nope, it uses the global WCT instance

File: .travis.yml (hunk 4 above)
Comment (stramel) [reply .travis.yml:12]: Already been added

File: .travis.yml (hunk 1 above)
Comment (stramel) [reply .travis.yml:15]: Not currently

File: mtz-url-rewrite-interceptor.html (hunk 2 above)
Comment (stramel) [reply mtz-url-rewrite-interceptor.html]: Nope that's correct.

Comment (stramel) [reply mtz-url-rewrite-interceptor.html]: I may go ahead and add them just for clarity but it should resolve correctly.

**Review by haydenmcp — APPROVED** [review]

**Pull request merged**